docs/rules/no-unsupported-node-api-in-ssrable-components.md

```diff
@@ -0,0 +1,28 @@
+# Disallow Node API Calls in SSR Context (`lwc/no-unsupported-node-api-in-ssrable-components`)
```

Comment:
```suggestion
# Disallow Node API Calls in SSR Context (`lwc/ssr/no-unsupported-node-api`)
```

Reply:
@lpomerleau , we can't change the path without changing the framework. So, I'm going to merge the PR without renaming.